CHANGELOG.md

```diff
@@ -11,8 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ### Changed
-
-- Changed all file line endings from CRLF to LF and added the .gitattributes to enforce it - [#163](https://github.com/DigitalExcellence/dex-backend/issues/163)
```

Comment:
Why did this line get removed?

Reply:
This looks like a merge conflict gone wrong ;)